charts/pihole/values.yaml

```diff
@@ -31,6 +31,9 @@ dnsHostPort:
 # -- Configuration for the DNS service on port 53
 serviceDns:
 
+  # -- deploys a mixed (TCP + UDP) Service instead of separate ones
+  mixedService: true
```

Comment:
Should we set this to `false` so everybody who want's to use it can opt in?

Reply:
changed and sorry, I intended to set it to `false` but must have forgotten.